UnityProject/Assets/Scripts/Objects/GasContainer.cs

```diff
@@ -156,11 +156,19 @@ private void ExplodeContainer()
 
 		public void ReleaseContentsInstantly()
 		{
-			MetaDataLayer metaDataLayer = registerObject.Matrix.MetaDataLayer;
-			MetaDataNode node = metaDataLayer.Get(registerObject.LocalPositionServer, false);
+			try
+			{
+				MetaDataLayer metaDataLayer = registerObject.Matrix.MetaDataLayer;
+				MetaDataNode node = metaDataLayer.Get(registerObject.LocalPositionServer, false);
 
-			GasMix.TransferGas(node.GasMix, GasMix, GasMix.Moles);
-			metaDataLayer.UpdateSystemsAt(registerObject.LocalPositionServer, SystemType.AtmosSystem);
+				GasMix.TransferGas(node.GasMix, GasMix, GasMix.Moles);
+				metaDataLayer.UpdateSystemsAt(registerObject.LocalPositionServer, SystemType.AtmosSystem);
+			}
+			catch (Exception e)
```

Comment:
was it throwing an NRE? You chould catch it specifically isntead of the generic Exception then